public_domain_review/utils.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from urllib.parse import urljoin
+
+import requests
+from bs4 import BeautifulSoup
+
+SOURCE_NAME = "public-domain-review"
+
+
+def get_outbound_links(url, html):
+    soup = BeautifulSoup(html, "html.parser")
+    links = soup.find_all("a")
+    outbound_links = [
```

Comment:
We should document that this code assumes all outbound links are relative links and not to like `www.example.com`. Also, that is a safe assumption right?

Reply:
Also maybe a comment about if `url` should have a trailing `/` as the behavior of urljoin changes with/without it https://stackoverflow.com/a/10893427